packages/node/src/index.ts

```diff
@@ -166,3 +166,5 @@ export type {
   User,
   Span,
 } from '@sentry/core';
+
+export type { FastifyRequest, ExpressRequest } from './integrations/tracing/nest/types';
```

Comment:
Is there a way we could get away without exporting these types? It's weird API surface.

Reply:
We could re-define them in the nest package?

Reply:
I would err towards doing that. Barely costs us anything and we spare ourselves breakage.